start_docker_service.sh

```diff
@@ -0,0 +1,24 @@
+HSUSER=headerservice
```

Comment:
Is there any reason to have to initialize this variable, in this script?

Reply:
Not for this script. I'll clean it up.